classes/patterns/class-pageheader.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Page Header pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Page Header.
+ *
+ * @package P4GBKS\Patterns
+ */
+class PageHeader extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/page-header';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 */
+	public static function get_config(): array {
+		$classname  = 'is-pattern-p4_page-header';
+		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
+		return [
+			'title'      => __( 'Page Header', 'planet4-blocks-backend' ),
+			'categories' => [ 'planet4' ],
+			'content'    => '
+				<!-- wp:cover {"url":"' . $media_link . '","dimRatio":0,"minHeight":550,"contentPosition":"top left","isDark":false,"align":"full","className":"' . $classname . '","style":{"color":{}}} -->
+				<div class="wp-block-cover alignfull is-light has-custom-content-position is-position-top-left ' . $classname . '" style="min-height:550px">
+					<span aria-hidden="true" class="has-background-dim-0 wp-block-cover__gradient-background has-background-dim"></span>
+					<img class="wp-block-cover__image-background" alt="Default image" src="' . $media_link . '" data-object-fit="cover"/>
```

Comment:
We should make the alt text a translatable string

Reply:
See [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/classes/patterns/class-quicklinks.php#L39) for example 🙂